### ISSUE TITLE
Export PLY with normals and curvature

### ISSUE THREAD

**Issue by animecomico** [cnr-isti-vclab/meshlab#148]
Hello,

When load a .ply with data of normals and curvature work fine, but, when export data, the curvature info is not exported.

**Comment by mcallieri** [comment]
Yes, at the moment, additional per-vertex and per-face data is not exported by MeshLab. not even in the PLY format.  :(
This behavior is questionable, I agree, and a more comprehensive exporting mechanism is on our to-do list.
Unfortunately, it will not be available in the immediate future.
